src/algorithms/acquisition/adapters/galileo_e1_pcps_ambiguous_acquisition.cc

```diff
@@ -17,272 +17,39 @@
-
-#if USE_GLOG_AND_GFLAGS
-    if (FLAGS_doppler_max != 0)
-        {
-            acq_parameters_.doppler_max = FLAGS_doppler_max;
-        }
-#else
-    if (absl::GetFlag(FLAGS_doppler_max) != 0)
-        {
-            acq_parameters_.doppler_max = absl::GetFlag(FLAGS_doppler_max);
-        }
-#endif
-    doppler_max_ = acq_parameters_.doppler_max;
-    doppler_step_ = static_cast<unsigned int>(acq_parameters_.doppler_step);
-    item_type_ = acq_parameters_.item_type;
-    item_size_ = acq_parameters_.it_size;
-    fs_in_ = acq_parameters_.fs_in;
-
-    code_length_ = static_cast<unsigned int>(std::floor(static_cast<double>(acq_parameters_.resampled_fs) / (GALILEO_E1_CODE_CHIP_RATE_CPS / GALILEO_E1_B_CODE_LENGTH_CHIPS)));
-    vector_length_ = static_cast<unsigned int>(std::floor(acq_parameters_.sampled_ms * acq_parameters_.samples_per_ms) * (acq_parameters_.bit_transition_flag ? 2.0 : 1.0));
-    code_ = volk_gnsssdr::vector<std::complex<float>>(vector_length_);
-
-    sampled_ms_ = acq_parameters_.sampled_ms;
-
-    DLOG(INFO) << "role " << role_;
-    acquisition_ = pcps_make_acquisition(acq_parameters_);
-    DLOG(INFO) << "acquisition(" << acquisition_->unique_id() << ")";
-
-    if (item_type_ == "cbyte")
-        {
-            cbyte_to_float_x2_ = make_complex_byte_to_float_x2();
-            float_to_complex_ = gr::blocks::float_to_complex::make();
-        }
-
-    if (in_streams_ > 1)
-        {
-            LOG(ERROR) << "This implementation only supports one input stream";
-        }
-    if (out_streams_ > 0)
-        {
-            LOG(ERROR) << "This implementation does not provide an output stream";
-        }
-}
-
-
-void GalileoE1PcpsAmbiguousAcquisition::stop_acquisition()
-{
-    acquisition_->set_active(false);
-}
-
-
-void GalileoE1PcpsAmbiguousAcquisition::set_threshold(float threshold)
-{
-    threshold_ = threshold;
-
-    acquisition_->set_threshold(threshold_);
-}
-
-
-void GalileoE1PcpsAmbiguousAcquisition::set_doppler_max(unsigned int doppler_max)
-{
-    doppler_max_ = doppler_max;
-
-    acquisition_->set_doppler_max(doppler_max_);
-}
-
-
-void GalileoE1PcpsAmbiguousAcquisition::set_doppler_step(unsigned int doppler_step)
-{
-    doppler_step_ = doppler_step;
-
-    acquisition_->set_doppler_step(doppler_step_);
-}
-
-
-void GalileoE1PcpsAmbiguousAcquisition::set_doppler_center(int doppler_center)
-{
-    doppler_center_ = doppler_center;
-
-    acquisition_->set_doppler_center(doppler_center_);
-}
-
-
-void GalileoE1PcpsAmbiguousAcquisition::set_gnss_synchro(Gnss_Synchro* gnss_synchro)
-{
-    gnss_synchro_ = gnss_synchro;
-
-    acquisition_->set_gnss_synchro(gnss_synchro_);
 }
 
 
-signed int GalileoE1PcpsAmbiguousAcquisition::mag()
+void GalileoE1PcpsAmbiguousAcquisition::set_channel(unsigned int channel)
 {
-    return acquisition_->mag();
+    cboc = configuration_->property("Acquisition" + std::to_string(channel) + ".cboc", false);
```

Comment:
Isn't this a bit odd, shouldn't it be `role + std::to_string(channel) + ".cboc"` perhaps?

Reply:
Even better, I did wonder why it used the channel number, was that an older way of setting parameters in the config?
I changed it and took the time to apply the same fix to all the other E1 acquisition classes.